### PR TITLE
Add pagination in News app

### PR DESCRIPTION
**Pull request by thomasdelhomenie**
This changes add pagination in the News app, using the Vuetify pagination component. It also changes the response of the GET /news endpoint to return the total size, offset and limit, as defined in the eXo REST guidelines.

### CODE REVIEW

File: webapp/src/main/webapp/news/components/NewsApp.vue [hunk 1]
```diff
@@ -102,19 +114,23 @@ export default {
...
       } else {
-        if (this.newsFilter === 'pinned') {
-          this.getPinnedNews();
-        } else {
-          this.getNewsList();
-        }
+        this.fetchNews(true);
       }
     },
     newsFilter() {
```
Comment (aelbenney) [inline webapp/src/main/webapp/news/components/NewsApp.vue]: With the refactoring done in getting news, we can call `fetcNews()` directly here when changing the filter, WDYT?

Comment (thomasdelhomenie) [reply webapp/src/main/webapp/news/components/NewsApp.vue]: It actually does not even work if we don't do that :smile: Good catch!

File: services/src/test/java/org/exoplatform/news/rest/NewsRestResourcesV1Test.java [hunk 2]
```diff
@@ -1419,7 +1425,7 @@ public void shouldGetUnauthorizedWhenSearchingWithTextInNonMemberSpaces() throws
     when(spaceService.isMember(anyString(),any())).thenReturn(false);
 
     //When
-    Response response = newsRestResourcesV1.getNews(request, "john", spacesIds, "published","pinned", text);
+    Response response = newsRestResourcesV1.getNews(request, "john", spacesIds, "published","pinned", text, 0, 10, false);
```
[the remark this answers is not in the document]
Comment (thomasdelhomenie) [reply services/src/test/java/org/exoplatform/news/rest/NewsRestResourcesV1Test.java]: I changed some existing tests and added some assertions. If it good for you ?

File: webapp/src/main/webapp/news/components/NewsApp.vue (hunk 1 above)
Comment (thomasdelhomenie) [reply webapp/src/main/webapp/news/components/NewsApp.vue]: Done

**Comment by thomasdelhomenie** [comment]
Replaced by https://github.com/exo-addons/news/pull/127

**Pull request closed without merging**